python-louvain/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
I didn't see Docker used in the readme. If this is used, we should include there and explanation of dependencies and how to run the container